packages/components/components/elvis-dropdown/src/react/dropdown-item/dropdownItem.tsx

```diff
@@ -178,8 +188,8 @@ export const DropdownItem: React.FC<DropdownItemProps> = ({
         aria-haspopup={item.children ? 'listbox' : 'false'}
         aria-expanded={isShowing}
         aria-selected={selfOrAllChildrenAreSelected}
-        aria-label={`${item.label}${item.children ? ' , undermeny' : ''}${
-          item.tooltip ? ' , Merknad: ' + item.tooltip : ''
+        aria-label={`${item.label}${item.children ? ` , ${labels.submenu}` : ''}${
+          item.tooltip ? ` , ${labels.note} ${item.tooltip}` : ''
```

Comment:
Be aware that this bit gives a "Refactor this code to not use nested template literals." on SonarCloud

Reply:
Fixed.